pkgs/servers/geospatial/geoserver/extensions.nix

```diff
@@ -29,325 +29,325 @@ in
 {
   app-schema = mkGeoserverExtension {
     name = "app-schema";
-    version = "2.24.2"; # app-schema
```

Comment:
@rolfschr ,

1. why all extensions versions are the same ?
2. why extensions versions change from `2.24.2` -> `2.25.2` 

Looks like a bug.

Reply:
I guess https://sourceforge.net/projects/geoserver/files/GeoServer/2.25.2/extensions/ is self explanatory.

Reply:
Thanks, I see, all extensions versions are really the same. But point nr. 2 looks like a problem. We should be updating extensions from `2.25.1 -> 2.25.2`.

Reply:
Indeed, some updates prior to #315582 are a bit problematic.

Reply:
Yes, as discussed [here](https://github.com/NixOS/nixpkgs/pull/314489) we previously did not correctly update the extensions while updating the package itself. This got fixed and now, for the first time, everything is updated the the same version.

Reply:
OK, so it should be fine for a next time. Thanks.